ckanext/apis/helpers.py

```diff
@@ -0,0 +1,17 @@
+import ckan.model as model
+from ckan.plugins import toolkit as tk
+
+c = tk.c
```

Comment:
I think tk.c was deprecated, and it's recommended to use tk.g nowadays (https://docs.ckan.org/en/2.9/extensions/plugins-toolkit.html#ckan.plugins.toolkit.ckan.plugins.toolkit.c)

Reply:
Implemented in [48e44f1](https://github.com/vrk-kpa/ckanext-apis/pull/7/commits/48e44f1abb34844a2b5579e0de5e99e033c81dd7)